LogAndStream/python_scripts/Bluetooth commands/s1_aAccel5Hz-sdbt.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+import struct
+import sys
+
```

Comment:
extra line.

Reply:
Pycharm sorts the imports like this when the file is 'tidied'